core/scss/components/card/_card.scss

```
@@ -0,0 +1,73 @@
+@charset "UTF-8";
+
+//
+// Card
+//
+// One of the primary building block compontents. This card component comes with
+// several fields to ensure flexibility.
+//
+// .su-card--horizontal - Postcard
+//
+// Markup: ../templates/components/card/card.twig
+//
+// Style guide: Components.Card
+//
+.su-card {
+  @include box-shadow('shallow');
+
+  > img {
+    // Hide the image on the smallest screen sizes.
+    @media #{$breakpoint-xs-only} {
+      display: none;
+    }
+  }
+}
+
+// Card contents spacing.
+.su-card__contents {
+  @include modular-margin;
+
+  // Ensure no bottom margins on last item.
+  *:last-child {
+    margin-bottom: 0;
```

Can we use the Bourbon margin mixin here for consistency?:

```suggestion
    @include margin(null null 0);
```